src/extension/providers/liquid/index.ts

```diff
@@ -25,12 +25,10 @@ export default class LiquidProvider {
   }
 
   async getAddress() {
-    await this.enable();
```

Comment:
shouldn't these have a check if it is enabled or not, and if not throw an exception (see the webln provider)

Reply:
yeah :see_no_evil: